.pre-commit-config.deployment.yaml

```diff
@@ -0,0 +1,10 @@
+# pre-commit for deployment related resources
```

Comment:
Or should this file be renamed `.pre-commit-config.requires-additional-binaries.yaml`, which might be more explicit as to why we have it in a separate file?